src/app/routes/utils/regex/utils/index.js

```diff
@@ -51,6 +51,11 @@ export const getLegacyAssetRegex = services => {
   return `/:service(${serviceRegex}):variant(${variantRegex})?/:assetUri(${legacyAssetUriRegex})?:amp(${ampRegex})?`;
 };
 
+export const getLiveRadioRegex = services => {
+  const serviceRegex = getServiceRegex(services);
+  return `/:service(${serviceRegex})/:serviceId(${mediaServiceIdRegex})/:mediaId(liveRadio):amp(${ampRegex})?`;
+};
+
 export const getRadioAndTVRegex = services => {
```

Comment:
Is it worth renaming this to onDemandRadioAndTVRegex now?

Reply:
The problem there would be that it would sound as though TV was also onDemand

Reply:
Let's leave it for now and come back to this whenever we tackle TV work.